setup.py

```diff
@@ -39,6 +39,7 @@
     "tabulate",
     "humanfriendly",
     "alembic",
+    "aiohttp",
```

Comment:
does it matter for us to include aiohttp's optional dependency `cchardet`?

Reply:
no, not used

Reply:
It's an optional dependency used in aiohttp https://github.com/aio-libs/aiohttp/blob/63a0d1040533947ef4973b3baed87e5fb7870e5a/aiohttp/client_reqrep.py#L69 but I wasn't sure about the performance implication

Reply:
Oh, I see. Maybe we can deal with it later with ujson.